shared/chat/conversation/attachment-get-titles/container.js

```diff
@@ -9,7 +9,10 @@ import {navigateUp} from '../../../actions/route-tree'
 import {type RouteProps} from '../../../route-tree/render-route'
 import type {PathToInfo} from '.'
 
-type OwnProps = RouteProps<{paths: Array<string>, conversationIDKey: Types.ConversationIDKey}, {}>
+type OwnProps = RouteProps<
+  {paths: Array<Types.PathAndOutboxID>, conversationIDKey: Types.ConversationIDKey},
```

Comment:
I feel like this should be changes to be called pathAndOutboxIDs instead of paths, to reflect the type better.